### ISSUE TITLE
Source files are published in the package

### ISSUE THREAD

**Issue by zsilbi** [metadevpro/openapi3-ts#134]
Hi!

When I open a project that uses this package, VSCode reports errors like the following:

```
Cannot write file 'node_modules/openapi3-ts/dist/dsl/openapi-builder30.d.ts' because it would overwrite input file.
```

The published package contains a `tsconfig.json` file with `compileOnSave": true` in it: https://www.npmjs.com/package/openapi3-ts/v/4.3.1?activeTab=code

I think these source files should be omitted when the package is published to npm.

**Comment by tarjelavik** [comment]
I am getting the same.

**Comment by RobinTail** [comment]
Made a fix for that, @tarjelavik && @zsilbi : #136
